mythril/laser/ethereum/instructions.py

```diff
@@ -710,11 +709,8 @@ def mstore_(self, global_state):
             # Attempt to concretize value
             _bytes = util.concrete_int_to_bytes(value)
 
-            i = 0
-
-            for b in _bytes:
-                state.memory[mstart + i] = _bytes[i]
-                i += 1
+            for i, b in enumerate(_bytes):
+                state.memory[mstart + i] = b
```

Comment:
we can assign a slice of `state.memory` as _bytes right? 
something like `state.memory[mstart: mstart+len(_bytes)] = _bytes`